### PR TITLE
connect URL

### PR DESCRIPTION
**Pull request by prakriti-solankey**

### CODE REVIEW

**Review by kartikpersistent — APPROVED** [review]

**Comment by prakriti-solankey** [comment]
1. The github icon

image

2. The Chatbot UI

image

File: docker-compose.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ services:
       - /app/node_modules
     environment:
       - BACKEND_API_URL=${BACKEND_API_URL-}
+      - PRODBLOOMURL = ${PROD_BLOOM_URL}
```
Comment (jexp) [inline docker-compose.yml]: it should just be one BLOOM_URL
that we configure differently for the two deployments

File: frontend/src/components/Content.tsx [hunk 2]
```diff
@@ -180,9 +180,15 @@ const Content: React.FC<ContentProps> = ({ isExpanded, showChatBot, openChatBot
     setShowAlert(false);
   };
 
-  const openGraphUrl = `https://bloom-latest.s3.eu-west-2.amazonaws.com/assets/index.html?connectURL=${
-    userCredentials?.userName
-  }@${localStorage.getItem('hostname')}%3A${localStorage.getItem('port') ?? '7687'}&search=Show+me+a+graph`;
+  const handleOpenGraphClick = () => {
+    const bloomUrl = process.env.BLOOM_URL;
+    const connectURL = `${userCredentials?.userName}@${localStorage.getItem('hostname')}%3A${
```
Comment (jexp) [inline frontend/src/components/Content.tsx:185]: misses the `neo4j+s://` prefix ??

File: frontend/src/components/Dropdown.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ const LlmDropdown: React.FC<DropdownProps> = ({ onSelect, isDisabled }) => {
   const handleChange = (selectedOption: OptionType | null | void) => {
     onSelect(selectedOption);
   };
-  const allOptions = useMemo(() => ['Diffbot', 'Gemini Pro'], []);
+  const allOptions = useMemo(() => ['Diffbot', 'OpenAI GPT 4'], []);
```
Comment (jexp) [inline frontend/src/components/Dropdown.tsx:9]: we should just send LLM to the backend and that uses that it's configuration to set the concrete LLM on the Source node.

**Review by jexp — APPROVED** [review]
small comments

File: docker-compose.yml (hunk 1 above)
Comment (prakriti-solankey) [reply docker-compose.yml]: Yes, it was by mistake added and forgot to clear it up. Thanks

File: frontend/src/components/Content.tsx (hunk 2 above)
Comment (prakriti-solankey) [reply frontend/src/components/Content.tsx:185]: It does not require the protocol according to what you shared here [neo4j%2B://<username>@<database-host>%3A<port>]. Also on implementation it redirects properly.

**Pull request merged**